apps/web/app/chatwoot/ChatwootWidget.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import { useCallback, useEffect, useRef } from "react";
+
+interface ChatwootWidgetProps {
+  chatwootBaseUrl: string;
+  isChatwootConfigured: boolean;
+  chatwootWebsiteToken?: string;
+  userEmail?: string | null;
+  userName?: string | null;
+  userId?: string | null;
+}
+
+export const ChatwootWidget = ({
+  userEmail,
+  userName,
+  userId,
+  isChatwootConfigured,
+  chatwootWebsiteToken,
+  chatwootBaseUrl,
+}: ChatwootWidgetProps) => {
+  const userSetRef = useRef(false);
+
+  const initializeChatwoot = useCallback(() => {
+    if (!isChatwootConfigured) {
```

Comment:
I would think that doing this check on the server makes much more sense instead of passing the whole component including data to the client to then decide not to render it.